src/export/validators.ts

```diff
@@ -0,0 +1,49 @@
+import { validateAuthor } from '../frontmatter/validators';
+import {
+  defined,
+  incrementOptions,
+  Options,
+  validateDate,
+  validateList,
+} from '../utils/validators';
+
+/**
+ * Validate Export Config
+ *
+ * TODO: This needs to be more complete, or even better, export config should more closely match
+ * project/site config and we can reuse those validators...
+ */
+export function validateExportConfigKeys(value: Record<string, any>, opts: Options) {
+  if (defined(value.data?.authors)) {
+    value.data.authors = validateList(
+      value.data.authors,
+      incrementOptions('authors', opts),
+      (author, index) => {
+        return validateAuthor(author, incrementOptions(`authors.${index}`, opts));
+      },
+    );
+  }
+  return value;
+}
+
+/**
+ * Perform validation/corsion on jtex output
+ *
+ * TODO: these coersions could remain in memory and be passed directly to jtex, instead of written to file
```

Comment:
Totally. Ideally jtex comes in to js land and we are writing less out to intermediate files.